src/swupd_comp_functions.c

```diff
@@ -136,6 +136,8 @@ int cmp_file_filename_is_deleted(const void *a, const void *b)
 	return 0;
 }
 
+#ifdef THIRDPARTY
```

Comment:
This is in the wrong commit.

Reply:
ups, that's true, I'll move it to the right commit